metadetect/detect.py

```diff
@@ -25,20 +25,39 @@
 
 class MEDSInterface(NGMixMEDS):
     """
-    Wrap a full image with a MEDS-like interface
+    Wrap a full image with a MEDS-like interface.
+
+    Parameters
+    ----------
+    obs: ngmix.observation.Observation
+        The observation with the image to wrap.
+    seg: np.array
+        The segmentation map for the observation.
+    cat: structured np.array
+        The catalog for the MEDS data format.
+    psf_rec: function, optional
```

Comment:
A function is accepted here since due to the pixel offsets, you can't simply pull the original psf reconstruction. We can pass around closures that encapsulate the original location etc.